src/app/gui/mainwindow.cpp

```diff
@@ -21,7 +21,7 @@
 
 #include "mainwindow.h"
 
-MainWindow::MainWindow(bool start_capture, bool enforce_affinity)
+MainWindow::MainWindow(bool start_capture, bool enforce_affinity, int num_cameras)
```

Comment:
:+1: on making this a CLI parameter! :-)

Reply:
Have you read my comment on the old PR? https://github.com/RoboCup-SSL/ssl-vision/pull/117#discussion_r259627279

Reply:
I see. Okay, that's fine for now.